conf/distro/nilrt.inc

```diff
@@ -70,6 +70,13 @@ SDK_VERSION := "${@'${DISTRO_VERSION}'}"
 LOCALCONF_VERSION = "1"
 LAYER_CONF_VERSION ?= "5"
 
+# The console specified here is used as default by start_getty in case a real console
+# isn't detected from /proc/consoles (happens when console is disabled from MAX).
+# If a console isn't detected, we don't want start_getty to exit as that would
+# cause sysvinit to respawn it.
+# Using tty2 here as it isn't used elsewhere.
+SERIAL_CONSOLES = "38400;tty2"
```

Comment:
Why does this need to be defined in `nilrt.inc`? Is it used outside of the `sysvinit-inittab` recipe?

Reply:
Well.. it doesn't have to be in `nilrt.inc`, we only need it for `sysvinit-inittab` recipe.
Other recipes that use it are `busybox-inittab`, `toybox-inittab`, `sysvinit-inittab`, `shadow-securetty`, `systemd-serialgetty`, but I don't believe we use any of these recipes.

Would you prefer I set it only for `sysvinit-inittab`?